src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -997,7 +997,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
 
         const auto modifiers = _GetPressedModifierKeys();
-        return _terminal->SendMouseEvent(terminalPosition, uiButton, modifiers, sWheelDelta);
+        return _terminal->SendMouseEvent(terminalPosition, uiButton, modifiers, sWheelDelta, { props.IsLeftButtonPressed(), props.IsMiddleButtonPressed(), props.IsRightButtonPressed() });

```

Comment:
I really don't like this, but constructing the state on a separate line means that I need to import VirtualTerminal::TerminalInput for just one line. Is there a way I can make this look nicer?

Reply:
Isn't that header already included though, because `TermControl.h` includes `Terminal.hpp` which includes `terminalInput.hpp`?

Reply:
Derp, didn't realize I was in a winrt namespace. Yeah, it works just fine :P